packages/node_modules/samples/browser-call-with-screenshare/app.js

```diff
@@ -81,30 +238,63 @@ function connect() {
           // This is just a little helper for our selenium tests and doesn't
           // really matter for the example
           document.body.classList.add('listening');
-          document.getElementById('connection-status').innerText = 'connected';
-          // Our device is now connected
-          resolve();
+          document.getElementById('register-status').innerText = 'device is registered';
+          enableMeetingActionButtons();
         })
         // This is a terrible way to handle errors, but anything more specific is
         // going to depend a lot on your app
         .catch((err) => {
+          console.log('error in connection');
           console.error(err);
-          // we'll rethrow here since we didn't really *handle* the error, we just
+          // We'll rethrow here since we didn't really *handle* the error, we just
           // reported it
           throw err;
         });
     }
-    else {
-      // Device was already connected
-      resolve();
-    }
+  });
+
+  //  Adding event handler for unregister button to unregister the device
+  document.getElementById('unregister').addEventListener('click', () => {
+    // again, we don't want to reload when we try to register
+    event.preventDefault();
+
+    webex.meetings.unregister();
+    disableMeetingActionButtons();
+    document.getElementById('register').disabled = false;
+    document.getElementById('register-status').innerText = 'device is not registered';
+    document.getElementById('auth-type').disabled = false;
+  });
+}
+
+// There's a few different events that'll let us know that we need to start listening
+// for incoming calls, so we'll wrap a few things up in a function.
+function connect() {
```

Comment:
Let's rename this for what it is actually doing since `connect` doesn't really apply any more.

```suggestion
function listenForIncomingMeetings() {
```